website/docs/concepts/contexts.mdx

```diff
@@ -91,11 +91,13 @@ The children are re-rendered when the context changes.
 
 ### Consuming context
 
-#### Struct components
-
-Use [HOC](../advanced-topics/struct-components/hoc)
-
 #### Function components
 
 `use_context` hook is used to consume contexts in function components.
 See [docs for use_context](function-components/hooks/use-context) to learn more.
+
+#### Struct components
+
+Use [HOC](../advanced-topics/struct-components/hoc)
```

Comment:
```suggestion
Use [Higher Order Components](../advanced-topics/struct-components/hoc)
```

I don't really like this section. Maybe it could be worded a little better?